internal/devices/devices.go

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
the commit message is nice but there's no need to be so formal. The previous informal style was fine (and actually I liked it a bit more).

Reply:
ack, good to know this feedback, I'll proceed with the informal style with the needed details :)